sdk/test-utils/test-credential/CHANGELOG.md

```diff
@@ -4,13 +4,13 @@
 
 ### Breaking Changes
 
-Updates the `createTestCredential` method to consume `DefaultAzureCredential` instead of `ClientSecretCredential` in order to offer autonomy to the devs and to move away from client secrets in environment varaibles.
+Updates the `createTestCredential` method to consume `ChainedTokenCredential` instead of `ClientSecretCredential` in order to offer autonomy to the devs and to move away from client secrets in environment varaibles.
```

Comment:
you'll have to make the same changes for the test-credential 1.x as hotfix/.

Reply:
Working on the hotfix here #30043 30043